types/src/keyless/openid_sig.rs

```diff
@@ -113,10 +114,10 @@ impl OpenIdSig {
     }
 
     /// `jwt_header` is the *decoded* JWT header (i.e., *not* base64url-encoded)
```

Comment:
update the doc too

Reply:
fudge, i'll have to do this later; don't wanna delay CI/CD (assuming this time it passes)

Reply:
fixed